test/requestContextPlugin.e2e.spec.js

```diff
@@ -0,0 +1,100 @@
+const fastify = require('fastify')
+const request = require('superagent')
+const { fastifyRequestContextPlugin } = require('../lib/requestContextPlugin')
+const { TestService } = require('./internal/testService')
+
+async function initAppPostWithPrevalidation(endpoint) {
```

Comment:
this function is duplicated in `test/requestContextPlugin.spec.js` (except ready vs listen)

Could you use once?

Reply:
Yeah, need to do a couple of deduplication passes. Will do.

Reply:
@Eomm Extracted this part, probably can extract some more.